pkgs/development/python-modules/torch/default.nix

```diff
@@ -240,6 +238,7 @@ buildPythonPackage rec {
       # Allow setting PYTHON_LIB_REL_PATH with an environment variable.
       # https://github.com/pytorch/pytorch/pull/128419
       ./passthrough-python-lib-rel-path.patch
+      ./0001-cmake.py-propagate-cmakeFlags-from-environment.patch
```

Comment:
I tried to rebuild this, and missed the patch.

Reply:
Thanks! Could you try again?